.editorconfig

```diff
@@ -14,6 +14,6 @@ insert_final_newline = true
 charset = utf-8
 
 # 4 space indentation
-[*.py, *.js, *.ts, *.tsx, *.css, *.scss, *.json, *.yaml, *.yml]
-indent_style = tab
-indent_size = 1
+[*.py, *.js, *.ts, *.tsx, *.css, *.scss, *.json, *.yaml, *.yml, *.scss, *.css]
+indent_style = space
+indent_size = 2
```

Comment:
Why? I use tabs

Reply:
i had been trying to make eslint formatter work, so it was changed during this process. For some reason my VSCode wasn't able to format JSX with tabs, it doesn't matter what to use i think

Reply:
So now all working only with spaces?